erddap/content/datasets.xml

```diff
@@ -2536,8 +2623,9 @@
       <destinationName>sensor</destinationName>
       <dataType>String</dataType>
       <addAttributes>
-        <att name="ioos_category">Unknown</att>
+        <att name="ioos_category">Other</att>
         <att name="long_name">Sensor</att>
+        <att name="standard_name">platform_name</att>
```

Comment:
There may not be an appropriate standard name for sensor, platform_name seems too broad.  Perhaps something else to submit to the CF repo.  In the meantime it may be better to have nothing in this space for the time being.

Reply:
https://dev.cioosatlantic.ca/erddap/categorize/long_name/sensor/index.html?page=1&itemsPerPage=1000

I had based it on the CF description, but am happy to remove the standard name if the CF definition doesn't fit? Maybe they only mean this in a different context?

platform_name
alias: station_description
"A variable with the standard name of platform_name contains strings which help to identify the platform from which an observation was made. For example, this may be a geographical place name such as "South Pole" or the name of a meteorological observing station. A "platform" is a structure or vehicle that serves as a base for mounting sensors. Platforms include, but are not limited to, satellites, aeroplanes, ships, buoys, **instruments**, ground stations, and masts."
http://cfconventions.org/Data/cf-standard-names/current/build/cf-standard-name-table.html

My rationale was moving this from the "Unknown" category to the "Other" category to try an eliminate all unknowns, but went a step farther and applied a general platform component, maybe there is a better way to refer to these?

Internally in these 10 datasets there is a dropdown list for these sensors, of three primary types, but the list itself was about a page long. Should we have three types entered in this field? The three types were aquameasure, hobo, and vr2ar, with a variety of numbers to identify each sensor as unique in some fashion. Here's one set as an example, see picture for reference.
https://dev.cioosatlantic.ca/erddap/tabledap/x9dy-aai9_split.html

![image](https://user-images.githubusercontent.com/28519105/180516732-3aca2f4d-dc76-4adb-b783-65f9a3c03b5c.png)

Reply:
Can simply remove the standard name too. Is there ever the case someone would want to filter on sensors looking for a certain type of easily comparable data for an apples to apples comparison (more than just say temperature or salinity would provide?).

Reply:
The CF description is quite broad and having taken a closer look at the sensors in question, they may very well qualify as platforms in their own right given their apparently self-contained nature.